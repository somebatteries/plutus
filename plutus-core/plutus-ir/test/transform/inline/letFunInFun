(let
    (nonrec)
    (termbind
      (strict)
      (vardecl idFun (fun (con integer) (con integer)))
      (lam x (con integer) x)
    )
    idFun
)