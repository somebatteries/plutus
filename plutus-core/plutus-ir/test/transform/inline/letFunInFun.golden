(lam x (con integer) x)